### PR TITLE
[TIMOB-23193] Implement target agnostic build options

### PR DESCRIPTION
**Pull request by infosia**
[TIMOB-23193](https://jira.appcelerator.org/browse/TIMOB-23193)

Rename following options:

- ws-cert → win-cert
- wp-sdk → win-sdk
- wp-product-guid → win-product-guid

This PR doesn't remove existing options. We might want to deprecate these options in `6.1.0` and remove them in next major release `7.0.0`.

```
// testing ws-cert and win-cert
appc run -p windows --target dist-winstore -l trace --ws-cert generated.cer
appc run -p windows --target dist-winstore -l trace --win-cert generated.cer

// testing wp-sdk and win-sdk works with 8.1
appc run -p windows --build-only --target wp-device -l trace --wp-sdk 8.1
appc run -p windows --build-only --target wp-device -l trace --win-sdk 8.1

// testing wp-sdk and win-sdk works with 10.0
appc run -p windows --build-only --target wp-device -l trace --wp-sdk 10.0
appc run -p windows --build-only --target wp-device -l trace --win-sdk 10.0

// testing wp-sdk and win-sdk works with specific 10.0 version
appc run -p windows --build-only --target wp-device -l trace --wp-sdk 10.0.10586.0
appc run -p windows --build-only --target wp-device -l trace --win-sdk 10.0.10586.0

// testing wp-local
appc run -p windows --build-only --target wp-local -l trace --wp-sdk 8.1
appc run -p windows --build-only --target wp-local -l trace --win-sdk 8.1
appc run -p windows --build-only --target wp-local -l trace --wp-sdk 10.0
appc run -p windows --build-only --target wp-local -l trace --win-sdk 10.0

// testing wp-emulator
appc run -p windows --build-only --target wp-emulator -l trace --wp-sdk 8.1
appc run -p windows --build-only --target wp-emulator -l trace --win-sdk 8.1
appc run -p windows --build-only --target wp-emulator -l trace --wp-sdk 10.0
appc run -p windows --build-only --target wp-emulator -l trace --win-sdk 10.0

// win-product-guid
appc run -p windows --build-only --target wp-device -l trace --wp-product-guid f0473be1-c557-4f98-a103-4ba9f453b5b0
appc run -p windows --build-only --target wp-device -l trace --win-product-guid f0473be1-c557-4f98-a103-4ba9f453b5b0
```

### CODE REVIEW

**Comment by infosia** [comment]
Hmm looks like Jenkins Windows 10 is building Windows 8.1 app. Investigating...

File: cli/commands/_build/config.js [hunk 1]
```diff
@@ -44,7 +46,7 @@ function config(logger, config, cli) {
 		supportedVisualStudioVersions: windowsPackageJson.vendorDependencies['visual studio'],
 		supportedWindowsPhoneSDKVersions: windowsPackageJson.vendorDependencies['windows phone sdk'],
 		tasklist: config.get('windows.executables.tasklist'),
-		skipWpTool: (cli.argv['target'] || cli.argv['T']) !== 'wp-device' || cli.argv['build-only']
+		skipWpTool: cli.argv['build-only'] || (target !== 'wp-device' && target !== 'wp-emulator')
```
Comment (infosia) [inline cli/commands/_build/config.js]: @garymathews 

Just found out that setting `skipWpTool:true` on `wp-emulator` target returns no emulators here. 
Introduced by #850 . So correct condition here should be:

- Always skip when `--build-only` is specified
- Do not skip when `--wp-device` is specified
- Do not skip when `--wp-emulator` is specified

**Steps to reproduce**

`appc run -p windows -T wp-emulator -l trace`

**Expected**

Emulators are listed

**Actual**
`No Windows Phone emulators found` exception fired.

Comment (garymathews) [inline cli/commands/_build/config.js]: Nice catch! 👍

**Comment by garymathews** [comment]
CR: **PASS**
FR: **PASS**

Looks good! 👍 

NOTE: `wp-local` should be `ws-local` in the test cases above

**Pull request merged**